src/BenchmarkDotNet/Configs/IConfig.cs

```diff
@@ -55,5 +56,7 @@ public interface IConfig
         IEnumerable<BenchmarkLogicalGroupRule> GetLogicalGroupRules();
 
         bool StopOnFirstError { get; }
+
+        CacheClearingStrategy CacheClearingStrategy { get; }
```

Comment:
important:: I think that this setting is very similar to Affinity and it would be nice to compare the results for the same benchmark with two different jobs.

So please move it from `IConfig` to `CacheClearingStrategyCharacteristic` in `EnvironmentMode`

https://github.com/dotnet/BenchmarkDotNet/blob/a76f438a81e3e0235a2e9b1249a5280038189689/src/BenchmarkDotNet/Jobs/EnvironmentMode.cs#L14

also, please don't forget to register a default value for it in the resolver